modules/shortcodes/polldaddy.php

```diff
@@ -45,7 +45,7 @@ private function get_async_code( array $settings, $survey_link ) {
 			'<div class="pd-embed" data-settings="'
 			. esc_attr( json_encode( $settings ) )
 			. '"></div>';
-		if ( $type === 'button' ) {
+		if ( $settings['type'] === 'button' ) {
```

Comment:
:yoda: style please!
`button === $settings['type']`

(same for below changeset)